### PR TITLE
gstreamer, gst_plugins_*: bump to 1.24.7

### PR DESCRIPTION
**Pull request by ElectrodeYT**
This change is in one commit as it must be done atomically. (at least, as far as I know)

### CODE REVIEW

**Comment by ElectrodeYT** [comment]
Rebased onto latest master and fixed the space/tab issues

**Comment by ElectrodeYT** [comment]
Added patch for `gst_plugins_base` to fix a install path for `gstglconfig.h`; didn't instantly notice it was installing into the lib folder

File: media-libs/gst_plugins_bad/gst_plugins_bad-1.24.7.recipe [hunk 1]
```diff
@@ -7,28 +7,31 @@ COPYRIGHT="Erik Walthinsen"
 LICENSE="GNU LGPL v2"
 REVISION="3"
```
Comment (korli) [inline media-libs/gst_plugins_bad/gst_plugins_bad-1.24.7.recipe:8]:
```suggestion
REVISION="1"
```

File: media-libs/gst_plugins_good/gst_plugins_good-1.24.7.recipe [hunk 2]
```diff
@@ -7,7 +7,7 @@ COPYRIGHT="Erik Walthinsen"
 LICENSE="GNU LGPL v2"
 REVISION="3"
```
Comment (korli) [inline media-libs/gst_plugins_good/gst_plugins_good-1.24.7.recipe:8]:
```suggestion
REVISION="1"
```

File: media-libs/gst_plugins_base/gst_plugins_base-1.24.7.recipe [hunk 3]
```diff
@@ -9,13 +9,13 @@ COPYRIGHT="Erik Walthinsen"
 LICENSE="GNU LGPL v2"
 REVISION="1"
 SOURCE_URI="https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-$portVersion.tar.xz"
-CHECKSUM_SHA256="f53672294f3985d56355c8b1df8f6b49c8c8721106563e19f53be3507ff2229d"
+CHECKSUM_SHA256="1528d1746a393299f5ac17ebf13a32a660202f1e29d0a852a2250f6a059a2fda"
 SOURCE_DIR="gst-plugins-base-$portVersion"
-
+PATCHES="gst_plugins_base-$portVersion.patchset"
 ARCHITECTURES="all !x86_gcc2"
```
Comment (korli) [inline media-libs/gst_plugins_base/gst_plugins_base-1.24.7.recipe:15]:
```suggestion
PATCHES="gst_plugins_base-$portVersion.patchset"

ARCHITECTURES="all !x86_gcc2"
```

File: media-libs/gst_plugins_good/gst_plugins_good-1.24.7.recipe (hunk 2 above)
Comment (Begasus) [reply media-libs/gst_plugins_good/gst_plugins_good-1.24.7.recipe:8]: Missing libX11 here in REQUIRES:
```
Warning: POLICY WARNING: "/packaging/gst_plugins_good/lib/gstreamer-1.0/libgstximagesrc.so" needs library "libX11.so.6", but the package doesn't seem to declare that as a requirement
```

**Comment by Begasus** [comment]
There is an order to build these, gstreamer for one is needed by the others, so should be build prior, should be looked into.

**Comment by Begasus** [comment]
Superseeded by: https://github.com/haikuports/haikuports/commit/ddc81967739f5fc0c934d9014bd76b7f84e66112

Maybe @threedeyes could have a look on the changes?

**Pull request closed without merging**